src/constants.js

```diff
@@ -1,4 +1,5 @@
 export const SITE_NAME = "Bento";
+export const ADMIN_NAME = "Admin";
```

Comment:
This probably doesn't need to be "variable-ized" the same way the site name does, since it won't really change

Reply:
This actually isn't needed. I removed the admin nav bar